test/credential.policies.e2e-spec.ts

```diff
@@ -247,9 +246,8 @@ describe('Issue and Prove credentials using policies (e2e)', () => {
             .send(invitation)
             .expect((res) => {
                 expect(res.status).toBe(200);
-                expect(res.body.results.length).toBeGreaterThan(0);
-                const i = res.body.results.length - 1;
-                expect(res.body.results[i].state).toBe('credential_acked');
+                expect(res.body.results.length).toBe(2);
```

Comment:
why are we rolling back these changes?

Reply:
these were accommodating to the changes in 5.6 acapy

Reply:
https://github.com/kiva/aries-guardianship-agency/commit/f332b30929503c102fdbf56cdc5242afe2014899